metrics.go

```diff
@@ -37,7 +37,7 @@ const (
 func (stats Stats) Metrics() map[string]any {
 	tags := make(map[string]any, len(stats.Timers)+len(stats.Truncations)+1)
 	for k, v := range stats.Timers {
-		tags[k] = uint64(v.Microseconds())
+		tags[k] = float64(v.Nanoseconds()) / float64(time.Microsecond) // The metrics should be in microseconds
```

Comment:
I'll note that making the division in int space should be faster... at the expense of losing fractionals (do we care?)

Reply:
Thats why it was an int before. Be at this point I think that backwards compatibility takes priority